src/platform/meteorlake/include/platform/lib/memory.h

```diff
@@ -0,0 +1,63 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Marcin Rajwa <marcin.rajwa@linux.intel.com>
+ *
+ */
+
+#ifdef __SOF_LIB_MEMORY_H__
+
+#ifndef __PLATFORM_LIB_MEMORY_H__
+#define __PLATFORM_LIB_MEMORY_H__
+
+#include <ace/lib/memory.h>
+
+#include <sof/lib/cpu.h>
+
+/* HP SRAM */
+#define HP_SRAM_BASE			0xA0020000
+
+/* HP SRAM windows */
+/* window 0 */
+#define SRAM_SW_REG_BASE		(HP_SRAM_BASE + 0x4000)
+#define SRAM_SW_REG_SIZE		0x1000
+
+#define SRAM_OUTBOX_BASE		(SRAM_SW_REG_BASE + SRAM_SW_REG_SIZE)
+#define SRAM_OUTBOX_SIZE		0x1000
+
+/* window 1 */
+#define SRAM_INBOX_BASE			(SRAM_OUTBOX_BASE + SRAM_OUTBOX_SIZE)
+#define SRAM_INBOX_SIZE			0x2000
+
+/* window 2 */
+#define SRAM_DEBUG_BASE			(SRAM_INBOX_BASE + SRAM_INBOX_SIZE)
+#define SRAM_DEBUG_SIZE			0x800
+
+#define SRAM_EXCEPT_BASE		(SRAM_DEBUG_BASE + SRAM_DEBUG_SIZE)
+#define SRAM_EXCEPT_SIZE		0x800
+
+#define SRAM_STREAM_BASE		(SRAM_EXCEPT_BASE + SRAM_EXCEPT_SIZE)
+#define SRAM_STREAM_SIZE		0x1000
```

Comment:
Above are not correct (should be in Zephyr side and not correct for IPC4 layout of SRAM).